src/test/java/com/ably/kafka/connect/DefaultChannelSinkMappingTest.java

```diff
@@ -0,0 +1,52 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DefaultChannelSinkMappingTest {
+    private DefaultChannelSinkMapping SUT;
+
+    //dependencies
+    private static final String STATIC_CHANNEL_NAME = "sink-channel";
+    private static ChannelSinkConnectorConfig STATIC_CHANNEL_CONFIG = new ChannelSinkConnectorConfig(Map.of("channel", STATIC_CHANNEL_NAME, "client.key", "test-key", "client.id", "test-id"));
+    private AblyRealtime ablyRealtime;
+
+    @BeforeEach
+    void setUp() {
+        SUT = new DefaultChannelSinkMapping(STATIC_CHANNEL_CONFIG);
+        try {
+            ablyRealtime = new AblyRealtime(STATIC_CHANNEL_CONFIG.clientOptions);
+        } catch (AblyException e) {
+            e.printStackTrace();
+        }
```

Comment:
Are we sure that we want to silently fail when we aren't able to create the `ablyRealtime`? :thinking: What will happen if the `ablyRealtime` is not created and then it gets used in one of the tests? Will we know that the cause of the test failure is this `AblyException`?

Reply:
That would have reflected to other tests with ablyRealtime being null, but I agree that's a less direct way to catch the exact reason. Fixed in https://github.com/ably/kafka-connect-ably/pull/40/commits/f3e2cd16c543c2a5a8b0418fdc622d6926a7f50f